### PR TITLE
Zl nav bar 2

### LINKED ISSUE #30: User Story 2, Visitor Navigation
```
As a visitor
I see a navigation bar
This navigation bar includes links for the following:
- a link to return to the welcome / home page of the application ("/")
- a link to browse all books ("/books")
```

### Checklist:

- [ ] base code tests are written
- [ ] base code is written to pass tests
- [ ] edge case ("sad path") tests written
- [ ] edge case code written
- [ ] code is reviewed
- [ ] branch is merged
- [ ] all tests still pass

child of #31

child of #33

### PR DESCRIPTION
**Pull request by zachlp2016**
__X_ Wrote Tests
__X_ Implemented
__X_ Reviewed

# Necessary checkmarks:
- [X] All Tests are Passing
- [X] The code will run locally

## Type of change
- [X] New feature
- [X] Bug Fix

# Implements/Fixes:

closes #30 

# Check the correct boxes
- [X] This broke nothing
- [] This broke some stuff
- [] This broke everything

# Testing Changes
- [] No Tests have been changed
- [X] Some Tests have been changed
- [] All of the Tests have been changed(Please describe what in the world happened)

# Checklist:

- [X] My code has no unused/commented out code
- [X] I have reviewed my code
- [] I have commented my code, particularly in hard-to-understand areas
- [X] I have fully tested my code

# Please include a link to a gif of how you feel about this branch:

### CODE REVIEW

File: app/assets/stylesheets/application.css [hunk 1]
```diff
@@ -10,6 +10,61 @@
...
  * It is generally better to create a new file per style scope.
  *
- *= require_tree .
- *= require_self
+ = require_tree .
+ = require_self
  */
+
+
+ *
```
Comment (blake-enyart) [inline app/assets/stylesheets/application.css:18]: We will add CSS reset file to replace this

File: app/views/pages/home.html.erb [hunk 2]
```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html>
+  <head>
```
Comment (blake-enyart) [inline app/views/pages/home.html.erb:3]: Add layout.erb

**Review by blake-enyart — APPROVED** [review]
